content/applications/websites/website/pages.rst

```diff
@@ -7,23 +7,22 @@ Pages
 Odoo allows you to create pages for your website and customize their content and appearance to your
 needs.
 
-.. note::
-   *Static* pages are pages that have stable content, such as the homepage. You can manually create
-   new ones, define their URL, adapt their :ref:`properties <website/page_properties>`, etc.
-   *Dynamic* pages, on the other hand, are generated dynamically. All pages generated automatically
-   by Odoo, for example when you install an app or module (e.g., `/shop` or `/blog`) or publish a
-   new product or blog post, are dynamic pages and are therefore managed differently.
+**Static** pages have stable content, such as the homepage. You can manually create new ones, define
```

Comment:
Didn't you want to add an anchor here so you could reference this paragraph elsewhere in tsi doc where we talk about static and dynamic pages?

Reply:
At first I thought it was clear enough as clarified in the intro but then I added the anchors